marimo/_snippets/data/duckdb-4.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 Marimo. All rights reserved.
+
+import marimo
+
+__generated_with = "0.11.0"
+app = marimo.App()
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+        # DuckDB: Parameterized & Reactive SQL Queries
+
+        This snippet shows how to parameterize SQL queries using reactive variables 
```

Comment:
marimo doesn't have "reactive variables", just variables

```suggestion
        This snippet shows how to parameterize SQL queries with Python variables
```